Foundation/NSPathUtilities.swift

```diff
@@ -574,6 +574,7 @@ extension FileManager {
 }
 
 public func NSSearchPathForDirectoriesInDomains(_ directory: FileManager.SearchPathDirectory, _ domainMask: FileManager.SearchPathDomainMask, _ expandTilde: Bool) -> [String] {
+    #if os(iOS)
```

Comment:
Then why do we have this everywhere?

#if os(OSX) || os(iOS)
    import Darwin
#elseif os(Linux)
    import Glibc
#endif

> On Aug 31, 2016, at 2:11 PM, Tony Parker <notifications@github.com> wrote:
> 
> In Foundation/NSPathUtilities.swift <https://github.com/apple/swift-corelibs-foundation/pull/614#discussion_r77074335>:
> 
> > @@ -574,6 +574,7 @@ extension FileManager {
> >  }
> >  
> >  public func NSSearchPathForDirectoriesInDomains(_ directory: FileManager.SearchPathDirectory, _ domainMask: FileManager.SearchPathDomainMask, _ expandTilde: Bool) -> [String] {
> > +    #if os(iOS)
> This doesn't quite make sense either - this code will only run on Linux.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/apple/swift-corelibs-foundation/pull/614/files/9a938f014ee096411f6e57236154a5e5bb75913f..ba7e298605cbb64c71e92a920f9de662279d9e09#r77074335>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AA15Cgb4UwRyTjMLnmE2PS6GGB3NZifcks5qle4IgaJpZM4JxoA6>.
>